### PR TITLE
feat: add LoadPolicyArray() to load policy from array

### PR DESCRIPTION
**Pull request by techoner**

### CODE REVIEW

File: persist/adapter.go [hunk 1]
```diff
@@ -37,10 +37,15 @@ func LoadPolicyLine(line string, m model.Model) {
 		return
 	}
 
-	key := tokens[0]
+	LoadPolicy(tokens, m)
+}
+
+// LoadPolicy loads a policy rule to model.
+func LoadPolicy(rule []string, m model.Model) {
```
Comment (tangyang9464) [inline persist/adapter.go:44]: Modify the method name to avoid confusion with the following interface

**Review by tangyang9464 — CHANGES_REQUESTED** [review]

File: persist/adapter.go (hunk 1 above)
Comment (techoner) [reply persist/adapter.go:44]: The scope of the two is different, I think it does not conflict.

Comment (tangyang9464) [reply persist/adapter.go:44]: Problems with code readability. Don't be ambiguous

**Review by nodece — APPROVED** [review]
LGTM

**Review by tangyang9464 — APPROVED** [review]
LGTM

**Pull request merged**